frame/asset-conversion/src/benchmarking.rs

```diff
@@ -55,7 +53,7 @@ where
 	let caller: T::AccountId = whitelisted_caller();
 	let caller_lookup = T::Lookup::unlookup(caller.clone());
 	if let Ok(asset_id) = T::MultiAssetIdConverter::try_convert(asset) {
-		assert_ok!(T::Currency::write_balance(&caller, BalanceOf::<T>::max_value()));
+		T::Currency::set_balance(&caller, u32::MAX.into());
```

Comment:
The `BalanceOf::<T>::max_value()` looked cleaner actually.

Reply:
yep, but setting it would result in having a zero balance, don't know why.

Reply:
`set_balance(&caller, BalanceOf::<T>::max_value())` fails since it tries to update the total_issuance under the hood that is already non-zero